### PR TITLE
Add NengoFPGA FPT paper

### PR DESCRIPTION
**Pull request by bmorcos**

### CODE REVIEW

File: ctn_waterloo/content/publications/morcos2018.bib [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                   Eliasmith, Chris and Kapre, Nachiket},
+    title       = {Implementing NEF Neural Networks on Embedded FPGAs},
+    booktitle   = {International Conference on Field-Programmable
+                   Technology (FPT)},
+    year        = {2018},
+    eventdate   = {2018-12-11/2018-12-14},
+    location    = {Naha, Okinawa, Japan},
+    publisher   = {IEEE},
+    pages       = {25-32},
+    doi         = {10.1109/FPT.2018.00015},
```
Comment (arvoelke) [inline ctn_waterloo/content/publications/morcos2018.bib]: Where did you get this doi from? This gives me not found: https://doi.org/10.1109/FPT.2018.00015 -- and googling it gives me a different paper about FPGAs.

File: ctn_waterloo/content/publications/morcos2018.bib [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    author      = {Morcos, Benjamin and Stewart, Terrence C and
+                   Eliasmith, Chris and Kapre, Nachiket},
+    title       = {Implementing NEF Neural Networks on Embedded FPGAs},
+    booktitle   = {International Conference on Field-Programmable
+                   Technology (FPT)},
+    year        = {2018},
+    eventdate   = {2018-12-11/2018-12-14},
+    location    = {Naha, Okinawa, Japan},
+    publisher   = {IEEE},
+    pages       = {25-32},
```
Comment (arvoelke) [inline ctn_waterloo/content/publications/morcos2018.bib]: Similarly where did you get this page range from?

File: ctn_waterloo/content/publications/morcos2018.bib (hunk 1 above)
Comment (bmorcos) [reply ctn_waterloo/content/publications/morcos2018.bib]: I got the DOI from the copy of the proceedings given to me at the conference, it hasn't been made available on the IEEE website yet. (the [2017 conference](https://ieeexplore.ieee.org/xpl/conhome.jsp?punumber=1000290) got added in Feb 2018... so not sure why it isn't up yet)

Screenshot from footer of the first page:
screenshot30

File: ctn_waterloo/content/publications/morcos2018.bib (hunk 2 above)
Comment (bmorcos) [reply ctn_waterloo/content/publications/morcos2018.bib]: Similarly, page range is from the copy of the proceedings I received at the conference.

File: ctn_waterloo/content/publications/morcos2018.bib (hunk 1 above)
Comment (arvoelke) [reply ctn_waterloo/content/publications/morcos2018.bib]: Resourceful! And yeah, weird that this isn't up yet.

**Pull request merged**